app/models/labware_creators/multi_stamp_tubes.rb

```diff
@@ -27,29 +27,30 @@ class MultiStampTubes < Base # rubocop:todo Style/Documentation
 
     validates :transfers, presence: true
 
-    PLATE_INCLUDES = 'wells,wells.aliquots,wells.aliquots.study'
-
     def allow_tube_duplicates?
       params.fetch('allow_tube_duplicates', false)
     end
 
     private
 
     def create_labware!
+      create_submission_from_parent_tubes
+
+      # TODO: Hack - change this to poll for status or something
+      puts '*** Waiting for it to build ***'
+      sleep(10)
```

Comment:
We need to fix this, although perhaps we can get it onto UAT to check the functionality is as expected in parallel